pmd-visualforce/etc/grammar/VfParser.jjt

```diff
@@ -146,7 +146,7 @@ PARSER_END(VfParser)
 	| <GE: ">=" >
 	| <LT: "<" >
 	| <GT: ">" >
-	| <EXCL: ("!"|"~") >
+	| <EXCL: ("!"|"~"|"NOT") >
```

Comment:
I'm not familiar with JavaCC, but is this case insensitive? Because 'Not' and 'not' will also work.